packages/ndla-ui/src/Resource/resourceComponents.tsx

```diff
@@ -90,35 +96,45 @@ const TagCounterWrapper = styled.p`
 
 interface TagListProps {
   tags?: string[];
+  tagLinkPrefix?: string;
 }
-
-export const TagList = ({ tags }: TagListProps) => {
+export const TagList = ({ tags, tagLinkPrefix }: TagListProps) => {
   if (!tags) return null;
   return (
     <StyledTagList>
       {tags.map((tag, i) => (
-        <StyledTagListElement key={`tag-${i}`}>{tag}</StyledTagListElement>
+        <StyledTagListElement key={`tag-${i}`}>
+          <StyledSafeLink
+            onClick={(e: MouseEvent<HTMLAnchorElement | HTMLElement>) => e.stopPropagation()}
+            to={`${tagLinkPrefix}/${tag}`}>
+            {tag}
+          </StyledSafeLink>
```

Comment:
No vil dette alltid rendres som ei lenke, men om komponenten brukes en anna plass uten tagLinkPrefix vil lenka bli for eksempel '/tag'. Usikker på om det er et problem. Du kunne eventuellt sjekka om tagLinkPrefix fantes før du rendra som lenke, og heller rendre som tekst dersom ikkje oppgitt.